dagmc_stats.py

```diff
@@ -300,3 +300,184 @@ def get_coarseness(my_core, meshset, entity_ranges, geom_dim):
         coarseness.append(len(surf_area)/sum(surf_area))
 
     return coarseness
+
+
+def get_beta_angles(my_core, pair_tris, common_vert, beta_angles):
+    """
+    Gets the beta angles of given triangle pairs
+
+    inputs
+    ------
+    my_core : a MOAB Core instance
+    pair_tris : triangle pair
+    common_vert : common vertices in the triangle pair
+    beta_angles: list that stores the beta angles of given triangle pair
+    """
+    vert1 = common_vert[0]
+    vert2 = common_vert[1]
+    side_lengths = []
+    
+    for m in range(2):
```

Comment:
Why not:
```suggestion
    for tri in pair_tris:
```
and corresponding changes below....